### PR TITLE
NA-32/Redesign header

### PR DESCRIPTION
**Pull request by yevgeny-semak**
image

### CODE REVIEW

**Comment by yevgeny-semak** [comment]
image
image

File: src/components/search-box.component.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import { IconButton, InputBase } from '@material-ui/core/';
+import SearchIcon from '@material-ui/icons/Search';
+import ClearIcon from '@material-ui/icons/Clear';
+
+import styled from 'styled-components';
+
+import { search } from '../redux/notesSlice';
+
+const SearchWrap = styled.div`
```
Comment (veranika-zakharchenia) [inline src/components/search-box.component.jsx:12]: wrapper

File: src/components/header.component.jsx [hunk 2]
```diff
@@ -2,51 +2,74 @@ import React, { useState } from 'react';
...
-    toolbarSearch: {
-        margin: "0 auto",
-    },
-    toolbarAccountIcon: {
-        color: "#ffffff",
-        fontSize: "3rem",
-    }
-}))
+const HeaderBar = styled(AppBar)`
+    background-color: #fb5;
```
Comment (veranika-zakharchenia) [inline src/components/header.component.jsx:23]: move colors to the constants

**Review by veranika-zakharchenia — CHANGES_REQUESTED** [review]
NB fix comments

File: src/components/search-box.component.jsx (hunk 1 above)
Comment (yevgeny-semak) [reply src/components/search-box.component.jsx]: fixed

File: src/components/header.component.jsx (hunk 2 above)
Comment (yevgeny-semak) [reply src/components/header.component.jsx]: fixed

**Pull request merged**